Tribler/Core/Modules/restapi/display_endpoint.py

```diff
@@ -77,10 +103,23 @@ def render_GET(self, request):
         :return: the node data formatted in JSON
         """
         if "focus_node" not in request.args:
-            return DisplayEndpoint.return_400(request, "focus_node parameter missing")
+            return DisplayEndpoint.return_error(request, message="focus_node parameter missing")
 
         if len(request.args["focus_node"]) < 1 or len(request.args["focus_node"][0]) == 0:
-            return DisplayEndpoint.return_400(request, "focus_node parameter empty")
+            return DisplayEndpoint.return_error(request, message="focus_node parameter empty")
```

Comment:
I would suggest a description like [this one](https://github.com/vandenheuvel/tribler/blob/master/Tribler/Core/Modules/restapi/downloads_endpoint.py#L254)